frameworks/elastic/tests/test_custom_options.py

```diff
@@ -0,0 +1,29 @@
+from tests import config
+import pytest
+import sdk_cmd
+import sdk_install
+import sdk_tasks
+import logging
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.sanity
+def test_xmx_and_xms_flags():
+    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+    #installing elastic service and passing customized json to overwrite default values.
+    sdk_install.install(
+        config.PACKAGE_NAME,
+        config.SERVICE_NAME,
+        config.DEFAULT_TASK_COUNT
+    )
+   #getting all the tasks and checking the flag duplicacy by running curl_cmd command.
+    for task in sdk_tasks.get_task_ids(config.SERVICE_NAME):
+        curl_cmd = "ps aux"
```

Comment:
The parameter being passed to `sdk_cmd.service_task_exec` was renamed to `cmd` below and not updated here.
```suggestion
        cmd = "ps aux"
```
Tests are breaking because of it: https://teamcity.mesosphere.io/viewLog.html?buildId=1729846&buildTypeId=DataServices_Elastic_CodeCheck&tab=buildLog